<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HITLOptionsPage</class>
 <widget class="QWidget" name="HITLOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>579</width>
        <height>593</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Choose flight simulator:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="chooseFlightSimulator"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Local interface (IP):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="hostAddress">
           <property name="toolTip">
            <string>For communication with sim computer via network. Should be the IP address of one of the interfaces of the GCS computer.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Remote interface (IP):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="remoteAddress">
           <property name="toolTip">
            <string>Only required if running simulator on remote machine. Should be the IP of the machine on which the simulator is running.</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Input Port:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="inputPort">
           <property name="toolTip">
            <string>For receiving data from sim</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Output Port:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="outputPort">
           <property name="toolTip">
            <string>For sending data to sim</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Path executable:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="Utils::PathChooser" name="executablePath" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Data directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="Utils::PathChooser" name="dataPath" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Attitude data</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>18</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="attRaw">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>AttitudeRaw (gyro, accels)</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Refresh rate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="attRawRate">
                 <property name="suffix">
                  <string>ms</string>
                 </property>
                 <property name="minimum">
                  <number>10</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>20</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="attActual">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>AttitudeActual</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QRadioButton" name="attActHW">
                 <property name="text">
                  <string>send raw data to board</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="attActSim">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>use values from simulator</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="attActCalc">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>calculate from AttitudeRaw</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Other data</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>18</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="homeLocation">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>HomeLocation</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Refresh rate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="homeLocRate">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="toolTip">
                  <string>0 - update once, or every N seconds</string>
                 </property>
                 <property name="suffix">
                  <string>sec</string>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="gpsPosition">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>GPSPosition</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Refresh rate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="gpsPosRate">
                 <property name="suffix">
                  <string>ms</string>
                 </property>
                 <property name="minimum">
                  <number>100</number>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>500</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="sonarAltitude">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>SonarAltitude</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="topMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Range detection</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="sonarMaxAlt">
                   <property name="suffix">
                    <string>m</string>
                   </property>
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>10</number>
                   </property>
                   <property name="value">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Refresh rate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="sonarAltRate">
                   <property name="suffix">
                    <string>ms</string>
                   </property>
                   <property name="minimum">
                    <number>20</number>
                   </property>
                   <property name="maximum">
                    <number>2000</number>
                   </property>
                   <property name="singleStep">
                    <number>10</number>
                   </property>
                   <property name="value">
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="inputCommand">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Map command from simulator</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>12</number>
               </property>
               <item>
                <widget class="QRadioButton" name="gcsReciever">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>to GCSReciver</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="manualControl">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>to ManualCtrll (not implemented)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>6</number>
              </property>
              <item>
               <widget class="QCheckBox" name="manualOutput">
                <property name="text">
                 <string>Maximum output rate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="outputRate">
                <property name="suffix">
                 <string>ms</string>
                </property>
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="value">
                 <number>15</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>chooseFlightSimulator</tabstop>
  <tabstop>hostAddress</tabstop>
  <tabstop>inputPort</tabstop>
  <tabstop>remoteAddress</tabstop>
  <tabstop>outputPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
